uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -376,10 +527,18 @@ sub determineScannerID {
     return $scannerID;
 }
 
-################################################################
-####### get_acqusitions($study_dir, \@acquisitions) ############ 
-####### puts list of acq dirs in @acquisitions #################
-################################################################
+
+=pod
+
+=head3 get_acqusitions($study_dir, \@acquisitions)
+
+Greps the list of acquisitions in the study directory from array
+C<@acquisitions>. ?????????????????????????????????????
```

Comment:
I don;t think this method is used anywhere.

Reply:
indeed, I just tried grepping for it in the LORIS-MRI code base and found nothing. Will just say unused function that will need to be removed in another PR.